lib/auth/assist/assistv1/test/service_test.go

```diff
@@ -358,36 +422,99 @@ func initSvc(t *testing.T) (map[string]context.Context, *assistv1.Service) {
 		ctxs[user.GetName()] = ctx
 	}
 
+	embedder := ai.MockEmbedder{
+		TimesCalled: make(map[string]int),
+	}
+
+	embeddings := &ai.SimpleRetriever{}
+	embeddingSrv := local.NewEmbeddingsService(backend)
 	svc, err := assistv1.NewService(&assistv1.ServiceConfig{
-		Backend:        local.NewAssistService(backend),
-		Authorizer:     authorizer,
-		Embeddings:     &ai.SimpleRetriever{},
-		ResourceGetter: &resourceGetterFake{},
+		Backend:    local.NewAssistService(backend),
+		Authorizer: authorizer,
+		Embeddings: embeddings,
+		ResourceGetter: &resourceGetterAllImpl{
+			PresenceService: presenceSvc,
+			AccessLists:     accesslistSvc,
+		},
+		Embedder: &embedder,
+	})
+	require.NoError(t, err)
+
+	unifiedResourcesCache, err := services.NewUnifiedResourceCache(context.Background(), services.UnifiedResourceCacheConfig{
```

Comment:
Do not pass context that never will be canceled. The background goroutine will leak
```suggestion
	unifiedResourcesCache, err := services.NewUnifiedResourceCache(ctx, services.UnifiedResourceCacheConfig{
```